src/app/components/addressAutoComplete/addressAutoComplete.tsx

```diff
@@ -0,0 +1,86 @@
+import { Combobox } from "@headlessui/react";
```

Comment:
Lets not have many folders in components. this can go to input folder

inputs -> This can be here
containers -> Buttons, Labels, Modals can be here